tasks/__init__.py

```diff
@@ -52,7 +52,7 @@ def test_module_wo_exit(ctx, module=None, numprocesses=1, params=['--reruns', '1
         numprocesses = cpu_count()
     # NOTE: Subprocess to compensate for lack of thread safety in the httpretty module.
     # https://github.com/gabrielfalcao/HTTPretty/issues/209#issue-54090252
-    args = ['-s', '-v', '--tb=short']
```

Comment:
Is it possible for -v and --tb=short to be used together? I like the old way of seeing each test listed out but I may just have fear of change.

Reply:
* `-v` / `--verbose` and `--tb=short` can be used together.
* I am for adding verbosity back. The dotted output is common practice in general but being able to see the current test and the list is very helpful in our selenium case.